RocketChat/Rocket.Chat.ReactNative
# Chore: Evaluate SidebarView - Typescript

## Proposed changes
Evaluate SidebarView

## Issue(s)	


## How to test or reproduce
On the app, open the side bar menu and check if everything still works fine. Also test the admin menu too.

## Screenshots

## Types of changes




## Checklist



## Further comments
